### PR TITLE
Got rid from Block implicitly warning

### PR DESCRIPTION
**Pull request by Maryom**
Got rid from Block implicitly retains 'self'; explicitly mention 'self' to indicate this is intended behavior

### CODE REVIEW

File: MGSwipeTableCell/MGSwipeTableCell.m [hunk 1]
```diff
@@ -253,30 +253,30 @@ -(void) expandToOffset:(CGFloat) offset settings:(MGSwipeExpansionSettings*) set
...
+                self->_expansionBackground.frame = [self expansionBackgroundRect:self->_expandedButton];
             }
-            else if (_expansionLayout == MGSwipeExpansionLayoutNone) {
-                [_expandedButton.superview bringSubviewToFront:_expandedButton];
-                _expansionBackground.frame = _container.bounds;
+            else if (self->_expansionLayout == MGSwipeExpansionLayoutNone) {
+                [self->_expandedButton.superview bringSubviewToFront:self->_expandedButton];
+                self->_expansionBackground.frame = self->_container.bounds;
             }
             else if (_fromLeft) {
```
Comment (battlmonstr) [inline MGSwipeTableCell/MGSwipeTableCell.m]: Forgot to change here.

Comment (Maryom) [reply MGSwipeTableCell/MGSwipeTableCell.m]: OK done ✅

**Comment by MortimerGoro** [comment]
Thanks for the updates, I'll review the PR soon. Sorry for the delay ;)

**Comment by Maryom** [comment]
NP :) You are welcome

**Pull request merged**